src/corehost/corehost.cpp

```diff
@@ -271,7 +243,7 @@ int run(const int argc, const pal::char_t* argv[])
         return StatusCode::InvalidArgFailure;
     }
 
-    app_root.assign(host_path);
+    app_root.assign(get_directory(host_path));
```

Comment:
This fixes a prior but benign bug; the app_root is not used with muxer but we still try to calculate it for consistency.